__tests__/app.test.js

```diff
@@ -166,6 +166,22 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 
+  test("POST 201: Respond with new added object to given article_id", () => {
+    const postComment = {
+      username: "butter_bridge",
+      body: "test-body",
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(postComment)
+      .expect(201)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body.comment[0].author).toBe("butter_bridge");
```

Comment:
Ideally we wouldn't have to access index 0, as theres only ever going to be 1 comment coming back from this request.